test/utils/url.test.js

```diff
@@ -0,0 +1,72 @@
+import { appendQuery } from 'src/utils';
+import nock from 'nock';
+import expect from 'expect';
+
+/**
+ * @private
+ * Adapted from https://github.com/lakenen/node-append-query
+ */
+describe('appendQuery', function(){
+    it('should append querystring to queryless url', function () {    
+        var result = appendQuery('http://example.com/foo', 'bar=baz&beep=boop')
+            , expected = 'http://example.com/foo?bar=baz&beep=boop'
```

Comment:
Hm, I don't know if you've had a look yet, but in the tests for `LiveService` I did used a more... visible scope. I feel it's more clear and you can always tell where it starts and ends, while I personally have no idea when and where the "this" refers to the same object in different describes and hooks. And if I'm overwriting something. I'd rather use the global scope than `this` as a global scope, if that's what's going on. I saw in the `test/index.js` file a client is set to `this` and then it's used in other places as `this.client` and was confused. Is that the best practice for mocha or something?